scripts/base.js

```diff
@@ -107,4 +107,53 @@ const updateCoordinates = async (
   return Ok("File updated successfully!");
 };
 
-export { Ok, Err, determineArgs, updateCoordinates };
+const updateParkingLots = async (
```

Comment:
This would be helpful to have some doctring describing what it does and the parameters